src/metabase/api/collection.clj

```diff
@@ -81,7 +81,7 @@
                                (collection/visible-collection-ids->honeysql-filter-clause
                                 :id
                                 (collection/permissions-set->visible-collection-ids @api/*current-user-permissions-set*))]
-                    :order-by [[:type :nulls-first] [:%lower.name :asc]]}) collections
+                    :order-by [[:type :asc] [:%lower.name :asc]]}) collections
```

Comment:
what does this change do?

Reply:
`:nulls-first` breaks this query on mysql. This preserves the proper ordering (instance_analytics should be last) and actually runs on mysql.